packit_service/worker/handlers/testing_farm.py

```diff
@@ -163,21 +163,26 @@ def _get_or_create_group(
         runs = []
         for target, build in builds.items():
             test_builds = [build] if build else []
-            runs.append(
-                TFTTestRunTargetModel.create(
-                    pipeline_id=None,
-                    identifier=self.job_config.identifier,
-                    commit_sha=self.data.commit_sha,
-                    status=TestingFarmResult.new,
-                    target=target,
-                    web_url=None,
-                    test_run_group=group,
-                    copr_build_targets=test_builds,
-                    # In _payload() we ask TF to test commit_sha of fork (PR's source).
-                    # Store original url. If this proves to work, make it a separate column.
-                    data={"base_project_url": self.project.get_web_url()},
+            if (
+                not self.testing_farm_job_helper.comment_arguments.identifier
+                or self.testing_farm_job_helper.comment_arguments.identifier
+                == self.job_config.identifier
```

Comment:
I would probably move this check into the [`checkers` for TF](https://github.com/packit/packit-service/blob/main/packit_service/worker/checker/testing_farm.py)  (probably create a new one, they are then listed [here](https://github.com/packit/packit-service/blob/d708d00beed1738c824f794c0e633a94d49e81f7/packit_service/worker/handlers/testing_farm.py#L119)) , if we checked it here, all the tests would already have the `The task was accepted..` status. If we create a check for this in checkers, the tasks for non-relevant job configurations will not be created